AdyenDropIn/Modules/Component/ComponentContainerViewModel.swift

```diff
@@ -61,17 +94,79 @@ internal class ComponentContainerViewModel: ComponentContainerViewModelProtocol
 extension ComponentContainerViewModel: PaymentComponentDelegate {
 
     func didSubmit(
-        _ data: Adyen.PaymentComponentData,
-        from component: any Adyen.PaymentComponent
+        _ data: PaymentComponentData,
+        from component: any PaymentComponent
     ) {
-        // TODO: -
-        print("SUBMIT")
+        let checkoutAttemptId = component.context.analyticsProvider?.checkoutAttemptId
+        let updatedData = data.replacing(
+            checkoutAttemptId: checkoutAttemptId
+        )
+
+        guard updatedData.browserInfo == nil else {
```

Comment:
I just realized something about having 2 submit functions in payment component and dropin. Adding a comment here to discuss offline more.